14/umbraco-cms/extending/database.md

```diff
@@ -4,7 +4,7 @@ description: A guide to creating a custom Database table in Umbraco
 
 # Creating a Custom Database Table
 
-It is possible to add custom database tables to your site to store additional data that should not be stored as normal content nodes.
+UmbracoCMS ships with [NPoco](https://github.com/schotime/NPoco), a lightweight, quasi-ORM that makes it easy to map the results of database queries to plain old CLR objects (POCOs). NPoco makes it possible to add custom database tables to your site to store additional data that should not be stored as normal content nodes.
```

Comment:
```suggestion
Umbraco ships with [NPoco](https://github.com/schotime/NPoco), which enables mapping the results of database queries to Common Language Runtime (CLR) objects. NPoco allows custom database tables to be added to your site to store additional data that should not be stored as normal content nodes.
```

Reply:
@sofietoft - Sorry I missed this notification! Your recommendations are great, thanks for getting this merged in!